CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [0.8.0] - 2022-04-01
+
```

Comment:
@MrAlias Awesome. Could you do a final check of the docs merge request and address Robert's concerns on TLS?